src/ResourceManager/Compute/Commands.Compute.Test/Commands.Compute.Test.csproj

```diff
@@ -243,6 +244,7 @@
     <None Include="ScenarioTests\RunnerTests.csv">
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </None>
+    <None Include="ScenarioTests\SqlIaaSExtensionTests.ps1" />
```

Comment:
@michaelcohenmicrosoft You need to add this, and the recorded mock to the csproj, and both should be marked as 'CopyAlways' or 'CopyIfNewer'